### PR TITLE
add dovetail 15

### PR DESCRIPTION
**Pull request by zkamvar**
This adds dovetail number 15, which announces the second stage of the beta phase. 

I will not be around on Friday to address reviews, but I will be able to do so on Monday.

@tobyhodges, would you kindly review this?

### CODE REVIEW

File: _posts/2023/02/2023-02-14-dovetail-15.md [hunk 1]
```diff
@@ -0,0 +1,223 @@
...
+<b><a href='https://carpentries.typeform.com/to/KRBl4IZM'>tell us about your experience</a>.</b></span> 
+
+---
+
+## Beta Phase Second Stage Is Live
+
+The second stage of the beta phase is now live. This is a big milestone because
+this is the portion of the beta phase where our community of contributors will
+get a chance to interact with and use the workbench in our live lessons before
+we transition all of our lessons to use The Workbench in May 2023.
```
Comment (tobyhodges) [inline _posts/2023/02/2023-02-14-dovetail-15.md]:
```suggestion
get a chance to interact with and use the Workbench in our live lessons before
we transition all of our lessons to use the new infrastructure in May 2023.
```

File: _posts/2023/02/2023-02-14-dovetail-15.md [hunk 2]
```diff
@@ -0,0 +1,223 @@
...
+
+With this second stage of the beta phase, we have created a series of videos that
+demonstrate some of the guides for what to expect during this period: 
+[Workbench Beta Phase YouTube Playlist](https://www.youtube.com/playlist?list=PLXLapl_LKb4eQ3xOFHpYEb15HF3-7cMPO).
+
+Here is the introduction to this stage of the Beta Phase:
```
Comment (tobyhodges) [inline _posts/2023/02/2023-02-14-dovetail-15.md]:
```suggestion
Here is the introduction to this stage of the beta phase: 
```

File: _posts/2023/02/2023-02-14-dovetail-15.md [hunk 3]
```diff
@@ -0,0 +1,223 @@
...
+demonstrate some of the guides for what to expect during this period: 
+[Workbench Beta Phase YouTube Playlist](https://www.youtube.com/playlist?list=PLXLapl_LKb4eQ3xOFHpYEb15HF3-7cMPO).
+
+Here is the introduction to this stage of the Beta Phase: 
+
+<iframe width="560" height="315" src="https://www.youtube-nocookie.com/embed/jnvEz-ojlUc" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share" allowfullscreen></iframe>
+
+On Monday, 2023-02-06, the beta phase lessons were transformed to
+enter the [second (beta) stage](https://carpentries.github.io/workbench/beta-phase.html#beta)
+of the beta phase: **one repository, two sites** where The Workbench version of
```
Comment (tobyhodges) [inline _posts/2023/02/2023-02-14-dovetail-15.md]:
```suggestion
of the beta phase: **one repository, two sites** where the Workbench version of
```

File: _posts/2023/02/2023-02-14-dovetail-15.md [hunk 4]
```diff
@@ -0,0 +1,223 @@
...
+the lesson is the default for the lesson repository. 
+
+For example, this is the state of the R for Social Scientists DataCarpentry lesson:
```
Comment (tobyhodges) [inline _posts/2023/02/2023-02-14-dovetail-15.md]:
```suggestion
For example, this is the state of the R for Social Scientists Data Carpentry lesson:
```

File: _posts/2023/02/2023-02-14-dovetail-15.md [hunk 5]
```diff
@@ -0,0 +1,223 @@
...
+| styles    | locked   | <https://datacarpentry.github.io/r-socialsci> | [datacarpentry/r-socialsci](https://github.com/datacarpentry/r-socialsci) | `legacy/gh-pages` |
+| Workbench | **live** | <https://preview.carpentries.org/r-socialsci> | [datacarpentry/r-socialsci](https://github.com/datacarpentry/r-socialsci) | `main`            |
+
+From now on, any changes to the lesson repository will only affect the
+Workbench version of the lesson. Because the git history of the default branch
+has changed, **if you have a fork of one of these lessons, you must [delete
+it and re-fork if you want to make a new contribution](https://carpentries.github.io/workbench/faq.html#update-fork-from-styles)**, otherwise, you 
+[risk a failed pull request](https://carpentries.github.io/sandpaper-docs/pull-request.html#transition-from-carpentriesstyles).
+
+On Monday, 2023-04-03 the default lesson website will use The Workbench and the
```
Comment (tobyhodges) [inline _posts/2023/02/2023-02-14-dovetail-15.md]:
```suggestion
On Monday, 2023-04-03 the default lesson website will use the Workbench and the
```

File: _posts/2023/02/2023-02-14-dovetail-15.md [hunk 6]
```diff
@@ -0,0 +1,223 @@
...
+styles branch will be removed from the GitHub repository.
+
+### Thank You Beta Phase Participants
+
+Being a maintainer is not an easy job. As I mentioned in my ode to Maintainers
+in dovetail 13, they are the quality assurance and moderators for our lessons
+to ensure that they adhere to our teaching principles and provide guidance for
+instructors and learners.
```
Comment (tobyhodges) [inline _posts/2023/02/2023-02-14-dovetail-15.md]:
```suggestion
Being a Maintainer is not an easy job. As I mentioned in my ode to Maintainers
in dovetail 13, they are the quality assurance and moderators for our lessons
to ensure that they adhere to our teaching principles and provide guidance for
Instructors and learners. 
```

File: _posts/2023/02/2023-02-14-dovetail-15.md [hunk 7]
```diff
@@ -0,0 +1,223 @@
...
+styles branch will be removed from the GitHub repository.
+
+### Thank You Beta Phase Participants
+
+Being a maintainer is not an easy job. As I mentioned in my ode to Maintainers
+in dovetail 13, they are the quality assurance and moderators for our lessons
+to ensure that they adhere to our teaching principles and provide guidance for
+instructors and learners. 
+
+The Maintainers who are participating in the Beta Phase have gone above and
```
Comment (tobyhodges) [inline _posts/2023/02/2023-02-14-dovetail-15.md]:
```suggestion
The Maintainers who are participating in the beta phase have gone above and
```

File: _posts/2023/02/2023-02-14-dovetail-15.md [hunk 8]
```diff
@@ -0,0 +1,223 @@
...
+beyond to test out The Workbench and provide valuable feedback that could only
```
Comment (tobyhodges) [inline _posts/2023/02/2023-02-14-dovetail-15.md]:
```suggestion
beyond to test out the Workbench and provide valuable feedback that could only
```

File: _posts/2023/02/2023-02-14-dovetail-15.md [hunk 9]
```diff
@@ -0,0 +1,223 @@
...
+
+ - [Jamie Jamison](https://github.com/jmjamison/)
+ - [Kaitlin Newson](https://github.com/kaitlinnewson/)
+ - [annaoates](https://github.com/annaoates/)
+
+
+I would also like to extend my sincere thanks to Saba Ferdous, Mark L Crowe,
+Sarah Kaspar, Jennifer Stubbs, Athanasia M Mowinckel, Jannetta Styen, 
+Sarah Stevens, and Philipp M Schäfer for their invaluable feedback, testing, and
+general enthusiasm for The Workbench.
```
Comment (tobyhodges) [inline _posts/2023/02/2023-02-14-dovetail-15.md]:
```suggestion
general enthusiasm for the Workbench.
```

File: _posts/2023/02/2023-02-14-dovetail-15.md [hunk 10]
```diff
@@ -0,0 +1,223 @@
...
+options(repos = c(
+  ropensci = 'https://carpentries.r-universe.dev',
+  CRAN = 'https://cloud.r-project.org'))
+# Download and install sandpaper in R
+install.packages(c('tinkr', 'pegboard', 'sandpaper', 'varnish'))
+```
+
+## Tips and Tricks for Using The Workbench
+
+One of the strengths of using The Workbench is that it gives you control over
```
Comment (tobyhodges) [inline _posts/2023/02/2023-02-14-dovetail-15.md]:
```suggestion
One of the strengths of using the Workbench is that it gives you control over
```

**Review by tobyhodges — APPROVED** [review]
Aside from some routine polishing changes, this looks great. After you've had a chance to work through my suggestions on Monday, I believe this can be merged.

**Comment by elletjies** [comment]
@zkamvar please let me know once this blog is ready to merge.

**Comment by zkamvar** [comment]
@elletjies This is ready to merge, Thank you!

**Pull request merged**